The build issue was introduced with 629a490cede4673cec29addd4629c432319a3b6f

  https://codereview.qt-project.org/c/qt/qtwebengine-chromium/+/537376

and not solved for us with 42f1f889315424f9341ea9b9cba645f66df8b0eb

  https://codereview.qt-project.org/c/qt/qtwebengine-chromium/+/539289

--- a/src/3rdparty/chromium/mojo/core/channel_posix.cc
+++ b/src/3rdparty/chromium/mojo/core/channel_posix.cc
@@ -254,7 +254,7 @@ class ChannelPosix : public Channel,
         socket_.reset();
         std::ignore = server_.TakePlatformHandle();
       }
- #if BUILDFLAG(OS_IOS)
+ #if defined(OS_IOS)
       base::AutoLock fd_lock(fds_to_close_lock_);
       fds_to_close_.clear();
  #endif
